src/routes/PassageDetail.tsx

```diff
@@ -188,7 +188,7 @@ const PassageDetailGrids = () => {
     setHeight(window.innerHeight);
     setDiscussionSize({
       width: discussionSize.width, //should we be smarter here?
-      height: window.innerHeight - 330,
+      height: window.innerHeight,
```

Comment:
I think the idea here was that 330 was being used by the other controls we wanted to keep on the screen. I guess removing this will require the screen to be scrolled.

Reply:
If we don't change it, we are guaranteeing they need to scroll the new discussion card every time (especially on a laptop)